frontend/src/komponenter/kopierLenke/KopierLenke.tsx

```diff
@@ -0,0 +1,37 @@
+import { Hovedknapp } from 'nav-frontend-knapper'
+import { Input } from 'nav-frontend-skjema'
+import React from 'react'
+import { Link } from '@navikt/ds-icons'
+
+export default function KopierLenke(): React.ReactElement {
+    const lenkeBase = 'www.innbyggerpanelet.no/'
+
+    // Lager en lenke med en tilfeldig ID på enden
+    const lenkeTekst = lenkeBase.concat(lagID(8))
+
+    // Lager en tilfeldig string
+    function lagID(lengde: number) {
+        let resultat = ''
+        const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+        const charactersLength = characters.length
+        for (let i = 0; i < lengde; i++) {
+            resultat += characters.charAt(Math.floor(Math.random() * charactersLength))
+        }
+        return resultat
+    }
```

Comment:
Er denne 100% random og ikke sudo-random? Tenkte bare i tilfelle får to personer samme link 😄